doc/nyan/engine_types.md

```diff
@@ -65,6 +68,7 @@ ResourceSpot():
 
 
 Building(Unit): # WIP: make another parent Object for Building?
+	# WIP2: Some buildings consist of a static frame and an animation on top. How do we reference that?
```

Comment:
Probably an `AnimatedBuilding`, also the building annexes like the towncenter need to be modeled out of parts.
They have an ordered set of sprites (which may be animated) so the draw order is fixed.